utils/lists.ts

```diff
@@ -276,6 +276,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Loot Craft is a brand new mechanism of resources 💎 and craft ⛏",
     url: "https://loot-craft.com/",
   },
+  {
+    name: "Cryptoodles Loot Bag",
```

Comment:
It seems that the website that you have added cannot be reached at this moment. Could you please check it and try to change and clarify the problem here?

Reply:
I can open the etherscan link. :)